src/DOM/rendering.ts

```diff
@@ -122,6 +122,7 @@ export function render(input: InfernoInput, parentDom?: Node | SVGAElement) {
 	if (devToolsStatus.connected) {
 		sendRoots(window);
 	}
+	return input;
```

Comment:
Doesn't React return the actual Component instance, or does it return a ReactElement (or VNode in Inferno's case)?